source/_integrations/androidtv.markdown

```diff
@@ -12,6 +12,12 @@ ha_codeowners:
 
 The `androidtv` platform allows you to control an Android TV device or [Amazon Fire TV](https://www.amazon.com/b/?node=8521791011) device.
 
+<div class='note info'>
```

Comment:
I think this note would be more fitting in the section where this is actually being instructed and not in the introduction text.